schema/site.schema.json

```
@@ -2567,6 +2567,14 @@
             "pointer": true
           },
           "default": "5000"
+        },
+        "excludeChunkOnError": {
+          "description": "Whether to cancel indexing a repo if an embedding request fails. If true, the chunk that cannot generate embeddings is not indexed and the remainder of the repository proceeds with indexing.",
```

small edit because I think it's not clear that "an embedding request" refers to a single file rather than a request to embed the repo.

```suggestion
          "description": "Whether to cancel indexing a repo if embedding a single file fails. If true, the chunk that cannot generate embeddings is not indexed and the remainder of the repository proceeds with indexing.",
```